### ISSUE TITLE
Transferring very large String-Properties

### ISSUE THREAD

**Issue by joha78** [canoo/dolphin-platform#420]
I implemented an upload/download mechanism via UUencoding binary data into a String and transferring it via a DolphinModel from server to client and vice versa. This works pretty fine. 

While using a DolphinAction for triggering loading on the server I do bind a progress wheel to AbstractViewBinder.actionInProcessProperty() and fetch the content on the server, putting it into the String property. Strange is that the progress wheels stops spinning before the String actually arrives in the client. I think it would be great if the model sync finishes before the action "comes back" from the server.

Furthermore it would be great if there would be a way to know that my large String finally arrived completely at the server.

**Comment by hendrikebbers** [comment]
@joha78 maybe you can add some "fake code". It would be good to see in what order you set the value to the model & call the action, etc.

**Comment by hendrikebbers** [comment]
@joha78 we already discussed this issue internally. So having a small code snippet would be cool. Based on this I could write a test

**Comment by joha78** [comment]
Now I am puzzled. Finally I put together a small project for demonstration but I can't reproduce the issue anymore. This might be because I created the demo with 0.10.0.  Next I will try my code with 0.10.0 and see if the problem disappeared too. Nevertheless, I attached the demo project for reference purposes...

[largestring.zip](https://github.com/canoo/dolphin-platform/files/873207/largestring.zip)

**Comment by timo0** [comment]
Thanks for your effort! Please let us know if the issue can be solved by update to version 0.10.0

**Comment by joha78** [comment]
Ok, I think I interpreted the behavior of my app wrong. I wrote an demonstration project that shows what happens:  https://github.com/joha78/dolphin-platform-large-string

1. Start the server & client
2. Press button 1 - this button loads a large image, converts it to a uu-encoded string and puts the string at the server into a model
3. Press button 2 - this button sets the same string field of the model at the server to a very short string. That takes a remarkable long time. 
4. Press button 2 again - now the server action comes back instantly. I think it should be the same with the first press at button 2?!?
